### PR TITLE
add fix for import tracker compatibility with py3.11

### PR DESCRIPTION
**Pull request by Fred-Fan**
https://github.com/IBM/import-tracker/issues/74

### CODE REVIEW

File: import_tracker/import_tracker.py [hunk 1]
```diff
@@ -322,9 +322,10 @@ def _get_value_col(dis_line: str) -> str:
 def _get_op_number(dis_line: str) -> Optional[int]:
     """Get the opcode number out of the line of `dis` output"""
     line_parts = dis_line.split()
-    if not line_parts:
+    temp = [i for i, val in enumerate(line_parts) if val.isupper()]
```
Comment (gabe-l-hart) [inline import_tracker/import_tracker.py:325]: Slight tightness suggestion (can't apply as a real `suggestion` over deleted lines I guess?)

```py
    if not (valid_line_part_idxs := [i for i, val in enumerate(line_parts) if val.isupper()]):
        return None
    opcode_idx = min(valid_line_parts)
```

**Review by gabe-l-hart — CHANGES_REQUESTED** [review]
Small code-golf improvement request. Also, it looks like you need to rebase with signoff:

```sh
remote=<remote name for maineline repo>
git fetch $remote
git rebase $remote/main --signoff
git push -f
```

File: import_tracker/import_tracker.py (hunk 1 above)
Comment (Fred-Fan) [reply import_tracker/import_tracker.py]: Updated. Thanks.

**Review by gabe-l-hart — APPROVED** [review]

File: import_tracker/import_tracker.py [hunk 2]
```diff
@@ -322,9 +322,9 @@ def _get_value_col(dis_line: str) -> str:
 def _get_op_number(dis_line: str) -> Optional[int]:
     """Get the opcode number out of the line of `dis` output"""
     line_parts = dis_line.split()
-    if not line_parts:
+    if not (valid_line_part_idxs := [i for i, val in enumerate(line_parts) if val.isupper()]):
```
Comment (gabe-l-hart) [inline import_tracker/import_tracker.py:325]:
```suggestion
    valid_line_part_idxs = [i for i, val in enumerate(line_parts) if val.isupper()]
    if not valid_line_part_idxs:
```

**Review by gabe-l-hart — CHANGES_REQUESTED** [review]
Let's try this

**Review by gabe-l-hart — APPROVED** [review]
Thanks for the fix!

**Pull request merged**